dcos_installer/async_server.py

```diff
@@ -341,7 +341,7 @@ def no_caching(request, response):
     # TODO(cmaloney): These should probably actually hard fail.
     try:
         app.router.add_static('/assets', assets_path)
-        app.router.add_static('/download/log', STATE_DIR)
+        app.router.add_static('/download/log', os.path.abspath(STATE_DIR))
```

Comment:
If we don't trust `add_static()` to correctly resolve relative paths, we should be doing this wherever we call it. Looks like the only other call is this other one above it, so please add it there as well.